### ISSUE TITLE
z14 - railway=level_crossing displayed on undisplayed railway=disused

### ISSUE THREAD

**Issue by matkoniecz** [gravitystorm/openstreetmap-carto#2074]
https://www.openstreetmap.org/#map=14/51.4687/21.7988
![selection_003](https://cloud.githubusercontent.com/assets/899988/13549739/c304f7fa-e30c-11e5-97a7-1f29916521ef.png)

It is rather confusing and not particularly useful.

**Comment by polarbearing** [comment]
I would consider that, at least in part, also a tagging error. When the railway is disused, it would also be a ```disused:railway=level_crossing```. I do not have to pay special attention anymore as there is not train to come.

**Comment by matkoniecz** [comment]
Note that railway=level_crossing marks place "where a road crosses a railway on the same level" - it is about structure itself. It implies possibility of passing trains but it is equally valid on line with 20 trains in each hour and in place where last train appeared a century ago.

**Comment by trigpoint** [comment]
> I would consider that, at least in part, also a tagging error. When the railway is disused, it would also > be a disused:railway=level_crossing. I do not have to pay special attention anymore as there is not 
> train to come. 
If the tracks are still present, then you may still have to slow down to cross comfortably.

**Comment by aceman444** [comment]
I also think when there is not a real crossing anymore (disused/abandoned railway) and you do not need to slow down much, the level_crossing should be removed or marked 'disused:*'. So that means if the mapper left level_crossing tagged, then it should also be rendered (not automatically hidden).

**Comment by ajank** [comment]
I would rather conclude that `railway=disused` should be rendered more prominently. According to the wiki, `railway=disused` implies that the track is still in place. Typically, the track will be rusty and might be covered by vegetation, but even such a disused track is a notable topographical feature. In my opinion, it should be rendered at z13+, i.e. the same level as `service=spur` railways.

Note that it might be impossible to distinguish between a rarely used and a disused railway track by just looking at it at the level crossing. You still have to slow down and beware of trains (although they are unlikely to come). To determine the track status, you have to check official registers or ask the locals who may tell something vague like "there was a cargo train every few months, but I have not seen one for a while" or even contradictory like "the track is disused, but there was a train recently".

**Comment by boothym** [comment]
I agree that level crossings on disused railways is a valid tagging combination, and should therefore be rendered at the same zoom level so they are visible together. 

Either move railway=disused up to z14, or level crossings down to z14 - I'd suggest the former, for the reasons mentioned by @ajank. It would also help make bridges carrying disused railways more prominent.

**Comment by boothym** [comment]
To move this issue along - here's an example near me at zoom 14 and 15, where you have level crossings shown on their own without a railway near them:

![untitled-4](https://user-images.githubusercontent.com/16965772/34462814-9c59b368-ee43-11e7-9996-46ae23e7a371.jpg)
![untitled-1](https://user-images.githubusercontent.com/16965772/34462815-9c70a28a-ee43-11e7-9236-5bbc7510d320.jpg)
This can be fixed in two ways, either:

- Level crossings are moved from z14 to z15
OR
- Disused railways are shown from z14 instead of z15

I would prefer the second option, as that would also help render bridges carrying disused railways, and make disused lines like the one through this part of the map more prominent: https://www.openstreetmap.org/#map=15/56.1833/-3.0772

Any chance someone could implement this?

**Comment by da1910** [comment]
Here's option two from above
![screenshot from 2019-02-01 16-36-28](https://user-images.githubusercontent.com/12086549/52136270-95d3ed80-263f-11e9-99b8-f8f663e90fa2.png)

**Comment by boothym** [comment]
@da1910 thanks. Would it possible to show a comparison screenshot between the current z14 and the new z14 - in a busier area with more than one disused line?

**Comment by da1910** [comment]
Here's prague with some disused lines, new z14, old z14 and z15 for comparison. It's a bit tricky because at z14 even in-use railway lines are pretty close together and pretty busy.
![screenshot from 2019-02-14 12-51-41](https://user-images.githubusercontent.com/12086549/52788246-41c20380-3058-11e9-8a68-b698f21a20b3.png)
![screenshot from 2019-02-14 12-57-19](https://user-images.githubusercontent.com/12086549/52788247-41c20380-3058-11e9-9767-d1840e26cfd8.png)
![screenshot from 2019-02-14 12-57-27](https://user-images.githubusercontent.com/12086549/52788248-41c20380-3058-11e9-81f7-2a8ed1110408.png)

**Comment by boothym** [comment]
Thanks, looks fine to me. Any chance of rendering this area nearby? https://www.openstreetmap.org/#map=14/50.0769/14.4892

**Comment by da1910** [comment]
No problem: 
![screenshot from 2019-02-15 14-07-37](https://user-images.githubusercontent.com/12086549/52861688-2b38ac80-312b-11e9-815c-1f5a69cd932c.png)

**Comment by da1910** [comment]
If we're happy with this solution I'll stick in a pull request and we can close this issue

**Comment by boothym** [comment]
I'm happy with the solution, though I don't get to decide so feel free to go ahead with the pull request and see what happens :). 

For the PR I think it would be good to have screenshots showing a proper before/after at z14 - if you can find an area of the map with disused railways and level crossing that would be great, or use the examples in the OP or my post later on.
